### DISCUSSION TITLE
MSNBC

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#392] [Show and tell]
### Channel Name

MSNBC

### Channel ID

MSNBC.us

### Website

None

### Stream URL

http://1128480543.rsc.cdn77.org/RXjs7q4qT52ExnvtXOJ_Xg==,1646411707/1128480543/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Tokened.

**Comment by jhedfors** [comment]
> Tokened.
@LaneSh4d0w I Been watching this ticket - can you explain what that means for this issue? Also should iptv-org/iptv#6470 be merged into this one, as once was broken, it is now gone?

**Comment by BellezaEmporium** [comment]
> > Tokened.
> @LaneSh4d0w I Been watching this ticket - can you explain what that means for this issue? Also should iptv-org/iptv#6470 be merged into this one, as once was broken, it is now gone?

This link needs a token to work. It changes every time and so needs to have a fresh one at every query.
